File: PHP/problem_1/algorithm.php
<?php

function sum($n, $k) {
  $d = floor($n / $k);
  return $k * (($d) * ($d + 1) / 2);
}

function main() {
    $n = 999;
    $s3 = sum($n, 3);
    $s5 = sum($n, 5);
    $s15 = sum($n, 15);

    $s = $s3 + $s5 - $s15;

    echo $s . "\n";
}

main();

File: Java/problem_1/Algorithm.java
class Algorithm {

    public static int sum(int n, int k) {
        return k * ((n / k) * (n / k + 1) / 2);
    }

    public static void main(String[] args) {
        int n = 999;
        int s3 = sum(n, 3);
        int s5 = sum(n, 5);
        int s15 = sum(n, 15);

        int s = s3 + s5 - s15;

        System.out.println(s);
    }

}
